pixi/src/checks/LinterCheck.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The AMPHTML Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const API_ENDPOINT = API_ENDPOINT_LINTER;
+
+export default class LinterCheck {
+  async run(pageUrl) {
+    const apiUrl = `${API_ENDPOINT}${pageUrl}`;
```

Comment:
pageUrl might contain parameters. it has to be URL escaped when you build the apiURL manually.
Or you do it the same way as the PageExperienceCheck